### ISSUE TITLE
rssfeed/server/dist/plugin-linux-amd64 not found

### ISSUE THREAD

**Issue by adunayevichg** [wbernest/mattermost-plugin-rssfeed#2]
I am running mattermost under docker.
When trying to enable the rssfeed I get that error message.
Do I need to install some additional software to install the plugin?

**Comment by wbernest** [comment]
No, you do not need to install any extra software.  If you can, turn your mattermost logging to DEBUG level and try to enable the plugin from the mattermost cli.  It should give you some better details about why mattermost is not copying the contents of the tar package to the directory.
